src/_includes/branch.njk

```diff
@@ -0,0 +1,24 @@
+---
+layout: base.njk
+---
+
+{{ content | safe}}
+
+<ul class="post-list">
+  {% for page in collections.web_dev %} 
```

Comment:
So this branch.njk iterates over just the web_dev collection? If we add another branch I guess this would break